flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/jdbc/JdbcPropertiesConverter.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.flink.connector.jdbc;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.table.catalog.ObjectPath;
+import org.apache.gravitino.flink.connector.PropertiesConverter;
+
+public abstract class JdbcPropertiesConverter implements PropertiesConverter {
+
+  private final Map<String, String> catalogOptions;
+
+  protected JdbcPropertiesConverter(Map<String, String> catalogOptions) {
+    this.catalogOptions = catalogOptions;
+  }
+
+  @Override
+  public Map<String, String> toGravitinoCatalogProperties(Configuration flinkConf) {
+    Map<String, String> gravitinoCatalogProperties =
+        PropertiesConverter.super.toGravitinoCatalogProperties(flinkConf);
+    if (!gravitinoCatalogProperties.containsKey(JdbcPropertiesConstants.GRAVITINO_JDBC_DRIVER)) {
+      gravitinoCatalogProperties.put(
+          JdbcPropertiesConstants.GRAVITINO_JDBC_DRIVER, defaultDriverName());
+    }
+    return gravitinoCatalogProperties;
+  }
+
+  @Override
+  public Map<String, String> toFlinkCatalogProperties(Map<String, String> gravitinoProperties) {
+    return PropertiesConverter.super.toFlinkCatalogProperties(gravitinoProperties);
+  }
+
+  @Override
+  public String transformPropertyToGravitinoCatalog(String configKey) {
+    return JdbcPropertiesConstants.flinkToGravitinoMap.get(configKey);
+  }
+
+  @Override
+  public String transformPropertyToFlinkCatalog(String configKey) {
+    return JdbcPropertiesConstants.gravitinoToFlinkMap.get(configKey);
+  }
+
+  @Override
+  public Map<String, String> toFlinkTableProperties(
+      Map<String, String> gravitinoProperties, ObjectPath tablePath) {
+    Map<String, String> tableOptions = new HashMap<>();
```

Comment:
If gravitinoProperties doesn't contain `driver`, should we return the driver from catalog properties?

Reply:
The flink JdbcCatalog will automatically infer the driver based on the URL.